packages/contracts-bedrock/src/deploy-utils.ts

```diff
@@ -276,3 +276,12 @@ export const getDeploymentAddress = async (
   const deployment = await hre.deployments.get(name)
   return deployment.address
 }
+
+export const jsonifyTransaction = (tx: ethers.PopulatedTransaction): string => {
+  return JSON.stringify({
+    to: tx.to,
+    data: tx.data,
+    value: '0',
```

Comment:
While none of the transactions send value, hardcoding this value could cause bugs for our future selves if things change in unexpected ways